lib/matplotlib/axes/_axes.py

```diff
@@ -1288,132 +1288,235 @@ def eventplot(self, positions, orientation='horizontal', lineoffsets=1,
+        Other Parameters
+        ----------------
+        scalex, scaley : bool, optional, default: True
+            These parameters determined if the view limits are adapted to
+            the data limits. The values are passed on to `autoscale_view`.
 
-        Neither line will be antialiased.
+        **kwargs : `~.Line2D` properties, optional
+            *kwargs* are used to specify properties like a line label (for
+            auto legends), linewidth, antialiasing, marker face color.
+            Example::
 
-        You do not need to use format strings, which are just
-        abbreviations.  All of the line properties can be controlled
-        by keyword arguments.  For example, you can set the color,
-        marker, linestyle, and markercolor with::
+            >>> plot([1,2,3], [1,2,3], 'go-', label='line 1', linewidth=2)
+            >>> plot([1,2,3], [1,4,9], 'rs',  label='line 2')
 
-            plot(x, y, color='green', linestyle='dashed', marker='o',
-                 markerfacecolor='blue', markersize=12).
+            If you make multiple lines with one plot command, the kwargs
+            apply to all those lines.
 
-        See :class:`~matplotlib.lines.Line2D` for details.
+            Here is a list of available `~.Line2D` properties:
 
-        The kwargs are :class:`~matplotlib.lines.Line2D` properties:
+            %(Line2D)s
 
-        %(Line2D)s
+        Returns
+        -------
+        lines
+            A list of `~.Line2D` objects that were added.
+
+
+        See Also
+        --------
+        scatter : XY scatter plot with markers of variing size and/or color (
+            sometimes also called bubble chart).
+
+
+        Notes
+        -----
+        **Format Strings**
+
+        A format string consists of a part for color, marker and line::
+
+            fmt = '[color][marker][line]'
+
+        Each of them is optional. If not provided, the value from the style
+        cycle is used. Exception: If ``line`` is given, but no ``marker``,
+        the data will be a line without markers.
+
+        **Colors**
+
+        The following color abbreviations are supported:
+
+        =============    ===============================
+        character        color
+        =============    ===============================
+        ``'b'``          blue
+        ``'g'``          green
+        ``'r'``          red
+        ``'c'``          cyan
+        ``'m'``          magenta
+        ``'y'``          yellow
+        ``'k'``          black
+        ``'w'``          white
+        =============    ===============================
+
+        In addition, you can specify colors in many weird and
+        wonderful ways, including full names (``'green'``), hex
+        strings (``'#008000'``), RGB or RGBA tuples (``(0,1,0,1)``) or
+        grayscale intensities as a string (``'0.8'``).  Of these, the
+        string specifications can be used in place of a ``fmt`` group,
```

Comment:
I would just say "In addition, colors can be specified using a Matplotlib `matplotlib.colors` specification."  If you want to add a few examples as well, thats fine, but a link to the colors module would help.